assignment2/commonAncestor_test.py

```diff
@@ -0,0 +1,77 @@
+"""Tests for commonAncestor."""
+
+from commonAncestor import commonAncestor
+from binarytree import BinaryTree
+import unittest
+
+class CommonAncestorTest(unittest.TestCase):
+    def setUp(self):
+        # Binary Search Tree
+        self.bst = BinaryTree(16)
+        self.bst.insert(9, 16, True)
+        self.bst.insert(18, 16, False)
+        self.bst.insert(19, 18, False)
+        self.bst.insert(14, 9, False)
+        self.bst.insert(3, 9, True)
+        self.bst.insert(1, 3, True)
+        self.bst.insert(5, 3, False)
+        if self.bst.getSize() != 8:
```

Comment:
Since this is a unittest, you want to actually call the `self.assertEqual` function instead.